File: src/main/java/com/atividade/plp/questao5/Aluno.java
package com.atividade.plp.questao5;

public class Aluno {

    private String name;
    private int n1;
    private int n2;
    private int n3;
    private int notaFinal;

    public String somarNota() {
        notaFinal = (n1 + n2 + n3);
        return "NOTA FINAL = " + notaFinal;
    }

    public String aprovadoOuReprovado() {
        if (notaFinal >= 60) {
            return "APROVADO";
        } else {
            return "REPROVADO";
        }
    }

    public String quantoFalta() {
        if (notaFinal < 60) {
            return "FALTARAM " + (60 - notaFinal);
        } else {
            return "";
        }
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getNota1() {
        return n1;
    }

    public void setNota1(int n1) {
        this.n1 = n1;
    }

    public int getNota2() {
        return n2;
    }

    public void setNota2(int n2) {
        this.n2 = n2;
    }

    public int getNota3() {
        return n3;
    }

    public void setNota3(int n3) {
        this.n3 = n3;
    }

    public int getNotaFinal() {
        return notaFinal;
    }

    public void setNotaFinal(int notaFinal) {
        this.notaFinal = notaFinal;
    }
}

File: C#/Questao5/Aluno.cs
using System;

namespace Questao5
{
    class Aluno
    {

        private string nome = "";
        private Double nota1 = 0;
        private Double nota2 = 0;
        private Double nota3 = 0;
        private Double notaFinal = 0;

        public Aluno()
        {
            //Construtor vazio
        }

        public string NOME 
        {
            get{return nome;}
            set{nome = value;}
        }

        public Double NOTA1
        {
            get{return nota1;}
            set{nota1 = value;}
        }

        public Double NOTA2
        {
            get{return nota2;}
            set{nota2 = value;}
        }

        public Double NOTA3
        {
            get{return nota3;}
            set{nota3 = value;}
        }

        public Double NOTA_FINAL
        {
            get{return notaFinal;}
            set{notaFinal = value;}
        }

        public String somarNota() 
        {
            notaFinal = (nota1 + nota2 + nota3);
            return "NOTA FINAL = " + notaFinal;
        }

        public String aprovadoOuReprovado() 
        {
            if (notaFinal >= 60) 
            {
                return "APROVADO";
            } else {
                return "REPROVADO";
            }
        }

        public String quantoFalta() 
        {
            if (notaFinal < 60) 
            {
                return "FALTARAM " + (60 - notaFinal);
            } else {
                return "";
            }
        }
    }
}
